### DISCUSSION TITLE
Log errors as `ERROR` to search them

### DISCUSSION THREAD

**Discussion by zirkelc** [biomejs/biome#4219] [Ideas]
Hi,

Biome logs errors with an✖ and warnings as  ⚠ :

![image](https://github.com/user-attachments/assets/8b0aeacc-e1d9-4444-8382-51bd850b0a05)

While that's nice see, it makes it unnecessary difficult to find the one error in a long list of warnings. It would be useful if the word `ERROR` actually appears somewhere in the log, for example next to the file name like `FIXABLE` does.

**Comment by ematipico** [comment]
If you aren't interested in warnings, you could use `--diagnostic-level=error`

**Reply by zirkelc** [reply]
I'm interested also in the warnings, but I need to fix the error and care about the warning later. So it would be useful to quickly find it  in the logs.
